src/commands/deploy.ts

```diff
@@ -219,17 +219,17 @@ export async function deployCommand(options: GenezioDeployOptions) {
                 options,
             ).catch(async (error) => {
                 if (error instanceof Error) {
-                    log.error(error.message);
                     if (error.message == "No frontend entry in genezio configuration file.") {
+                        log.error(error.message);
```

Comment:
We are still doing `exit(0)` here. Is this indended?

Reply:
there are still a few places where I need to enforce the new error handling mechanism, as well as integrate it with the Axios interceptor. I am working on this right now.